### PR TITLE
docs: Document the limitation between uib and app directory

### PR DESCRIPTION
**Pull request by benjaminParisel**
* when uib-app is connected to github, the app directory link is not working

### CODE REVIEW

File: modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc [hunk 1]
```diff
@@ -41,4 +41,20 @@ These resources guide you through managing Git integration, including handling m
...
+* xref:applications:resolve-merge-conflicts.adoc[Resolve merge conflicts in Git]
+
+
+[.troubleshooting-title]
+== Troubleshooting
+
+[.troubleshooting-section]
+--
+[.symptom]
+During development phase, from the application directory, I cannot see the living application.
```
Comment (marcobonita) [inline modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc]:
```suggestion
During development phase, from the Bonita application directory, I cannot open the application made with UI Builder (`Internal server error while processing request` error).
```

File: modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc [hunk 2]
```diff
@@ -41,4 +41,20 @@ These resources guide you through managing Git integration, including handling m
...
+
+[.symptom-description]
+You UI-Builder application is connected to Git repository. This links between the application and bonita application directory is not supported.
```
Comment (marcobonita) [inline modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc]:
```suggestion
Your UI Builder application is connected to a Git repository. This link between the application and the Bonita application directory is not yet supported.
```

File: modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc [hunk 3]
```diff
@@ -41,4 +41,20 @@ These resources guide you through managing Git integration, including handling m
...
+
+[.solution]#Possible solution#
+During development, you can use the UI-Builder application directory.
```
Comment (marcobonita) [inline modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc]:
```suggestion
During development phase, you can access the UI Builder application from the URL (e.g. `http://localhost:8090/app/your-application-name/…`).
```

File: modules/runtime/pages/application-directory.adoc [hunk 4]
```diff
@@ -37,6 +37,12 @@ To access its content:
 . In the menu, click on "Applications"
 . On the "Bonita Application Directory" row, click on the "..." _View application details_ icon
 
+WARNING:
+--
+On development phase, if you try to access to a xref:applications:ui-builder/version-control-with-git/version-control-with-git.adoc[UI-Builder application connected to Git repository] from the Application Directory, the redirection will fail.
+
+To access to the UI-Builder application, you need to access it directly from the Ui-Builder URL.
```
Comment (marcobonita) [inline modules/runtime/pages/application-directory.adoc]:
```suggestion
To access the UI Builder application, you need to access it directly from the URL (e.g. `http://localhost:8090/app/your-application-name/…`).
```

File: modules/runtime/pages/application-directory.adoc [hunk 5]
```diff
@@ -37,6 +37,12 @@ To access its content:
 . In the menu, click on "Applications"
 . On the "Bonita Application Directory" row, click on the "..." _View application details_ icon
 
+WARNING:
```
Comment (marcobonita) [inline modules/runtime/pages/application-directory.adoc]: L'affichage du warning est cassé sur la preview : 
image

**Review by marcobonita — CHANGES_REQUESTED** [review]

File: modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc (hunk 3 above)
Comment (benjaminParisel) [reply modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc]:
```suggestion
During development phase, you can access the UI Builder application from the default URL (e.g `http://localhost:8090/`)
```

File: modules/runtime/pages/application-directory.adoc (hunk 4 above)
Comment (benjaminParisel) [reply modules/runtime/pages/application-directory.adoc]:
```suggestion
To access the UI Builder application, you need to access it directly from the URL (e.g `http://localhost:8090/`)
```

File: modules/runtime/pages/application-directory.adoc (hunk 5 above)
Comment (benjaminParisel) [inline modules/runtime/pages/application-directory.adoc]:
```suggestion
[WARNING]
```

File: modules/runtime/pages/application-directory.adoc [hunk 6]
```diff
@@ -37,6 +37,12 @@ To access its content:
...
+--
```
Comment (benjaminParisel) [inline modules/runtime/pages/application-directory.adoc]:
```suggestion
==
```

File: modules/runtime/pages/application-directory.adoc [hunk 7]
```diff
@@ -37,6 +37,12 @@ To access its content:
...
+On development phase, if you try to access to a xref:applications:ui-builder/version-control-with-git/version-control-with-git.adoc[UI-Builder application connected to Git repository] from the Application Directory, the redirection will fail.
+
+To access to the UI-Builder application, you need to access it directly from the Ui-Builder URL.
+--
```
Comment (benjaminParisel) [inline modules/runtime/pages/application-directory.adoc]:
```suggestion
==
```

File: modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc [hunk 8]
```diff
@@ -41,4 +41,20 @@ These resources guide you through managing Git integration, including handling m
...
+[.troubleshooting-section]
+--
+[.symptom]
+During development phase, from the Bonita application directory, I cannot open the application made with UI Builder (`Internal server error while processing request` error).
+
+[.symptom-description]
+Your UI Builder application is connected to a Git repository. This link between the application and the Bonita application directory is not yet supported.
+
+[.solution]#Possible solution#
+During development phase, you can access the UI Builder application from the default URL (e.g `http://localhost:8090/`)
```
Comment (abirembaut) [inline modules/applications/pages/ui-builder/version-control-with-git/version-control-with-git.adoc]: what about ?
```suggestion
During development phase, you can access the UI Builder application from the UI Builder workspace using the default URL (e.g `http://localhost:8090/`)
```

File: modules/runtime/pages/application-directory.adoc [hunk 9]
```diff
@@ -37,6 +37,12 @@ To access its content:
 . In the menu, click on "Applications"
 . On the "Bonita Application Directory" row, click on the "..." _View application details_ icon
 
+[WARNING]
+====
+On development phase, if you try to access to a xref:applications:ui-builder/version-control-with-git/version-control-with-git.adoc[UI-Builder application connected to Git repository] from the Application Directory, the redirection will fail.
+
+To access the UI Builder application, you need to access it directly from the URL (e.g `http://localhost:8090/`)
```
Comment (abirembaut) [inline modules/runtime/pages/application-directory.adoc]:
```suggestion
To access the UI Builder application, you need to access it directly from the U Builder workspace (e.g `http://localhost:8090/`)
```

File: modules/runtime/pages/application-directory.adoc [hunk 10]
```diff
@@ -37,6 +37,12 @@ To access its content:
 . In the menu, click on "Applications"
 . On the "Bonita Application Directory" row, click on the "..." _View application details_ icon
 
+[WARNING]
+====
+On development phase, if you try to access to a xref:applications:ui-builder/version-control-with-git/version-control-with-git.adoc[UI-Builder application connected to Git repository] from the Application Directory, the redirection will fail.
```
Comment (abirembaut) [inline modules/runtime/pages/application-directory.adoc]:
```suggestion
During the development phase, if you try to access a xref:applications:ui-builder/version-control-with-git/version-control-with-git.adoc[UI-Builder application connected to a Git repository] from the Application Directory, the redirection will fail.
```

**Review by abirembaut — COMMENTED** [review]
Just a fex suggestions

**Review by abirembaut — APPROVED** [review]

**Pull request merged**